packages/wrangler/src/tail/printing.ts

```diff
@@ -48,6 +50,10 @@ export function prettyPrintLogs(data: WebSocket.RawData): void {
 		).toLocaleString();
 
 		logger.log(`Alarm @ ${datetime} - ${outcome}`);
+	} else if (isTailInfo(eventMessage.event)) {
```

Comment:
A lot of `else if`'s starting to getting added, we may want to start thinking about the code design here.

Reply:
Maybe a switch/reducer kind of pattern, or something to define the `event` state.